src/display/msgtree.rs

```diff
@@ -188,7 +188,7 @@ impl<'r, I> Iterator for TreeGraphElemLineIterator<'r, I>
             let mut parent_update = commit.parent(0).as_ref().map(Commit::id).ok();
 
             // generate graph elements for the parents currently tracked
-            let mut elems : TreeGraphElemLine = self.parents.iter_mut().map(|mut parent| {
+            let mut elems : TreeGraphElemLine = self.parents.iter_mut().map(|parent| {
```

Comment:
Not sure about this one. It might be one of those cases where people didn't consider the implications of semantic versioning, e.g. it is mandatory if a specific version of $whatever is used.